File: Front-End-Rincon/ElRinconDeLasCartas/src/app/model/cartaCarrito.ts
export class cartaCarrito2 {
    carritoFId: number;
    cartaVendedorFId: number;
    fCantidad: number;
}

export class cartaCarrito {
    cartaCarritoId: number;
    carritoFId: number;
    cartaVendedorFId: number;
    fCantidad: number;
}

export class cartaCarritoAllData {
    cartaCarritoId: number;
    carritoFId: number;
    cartaVendedorFId: [{
        CartaFName: String;
        Precio: number;
    }];
    fCantidad: number;
}

File: Back-End-Rincon/RinconDL/Entities/CartaCarrito.cs
using System.Collections.Generic;

namespace RinconDL.Entities
{
    public class CartaCarrito
    {
        public int CartaCarritoId { get; set; }
        public int CarritoFId { get; set; }
        public int CartaVendedorFId { get; set; }
        public int FCantidad { get; set; }
    }
}
